recipes/hicstuff/meta.yaml

```diff
@@ -31,6 +31,9 @@ requirements:
     - requests
     - scipy
     - scikit-learn
+    - minimap2 
```

Comment:
@lecorguille they are listed as external dependencies: https://github.com/koszullab/hicstuff#external-dependencies

Should that not be better handled in a requirements file or section?

Reply:
IMHO, not so "external" : https://github.com/koszullab/hicstuff/blob/f9cad7c136e76713de0328fdd37201d579ce6380/hicstuff/iteralign.py#L156-L165